src/Toolkit/Toolkit.iOS/UI/Controls/ScaleLine/ScaleLine.iOS.cs

```diff
@@ -210,7 +214,11 @@ private void Initialize()
         private void ScaleLine_PropertyChanged(object sender, PropertyChangedEventArgs e)
         {
             // Update the scale line to be the longer of the metric or imperial lines
-            _combinedScaleLine.Width = _metricScaleLine.Width > _usScaleLine.Width ? _metricScaleLine.Width : _usScaleLine.Width;
+            if (_combinedScaleLine != null && _metricScaleLine != null && _usScaleLine != null)
```

Comment:
Are these null-checks needed? Looks like `ScaleLine_PropertyChanged` handler is not added until AFTER all of these fields are set.